### ISSUE TITLE
AJS: Restrict possible loadout to accepted variants

### ISSUE THREAD

**Issue by vanosten** [NikolaiVChr/flightgear-saab-ja-37-viggen#111]
Cf. Fpl AJS 37 speciell förarinstruktion (SFI) del 2.pdf, page 188 in Kap IV. E.g. the use of the outermost pylons is quite restricted.

**Comment by colingeniet** [comment]
That's a nice table!
Flightgear-wise there are two questions regarding loadout: the `Fuel and Payload` menu, and the quick payload select menus (in `AJS-37->Reload...`).
I think restricting the former would be excessive. However the payload quick select could certainly use a rework. I propose the following:
- Add a separate payload quick select window (like the f16), instead of the various existing 'reload ...' menu entries.
- Populate it with some of the most 'interesting' loadout options from that table

**Comment by vanosten** [comment]
Agree with a separate payload quick select. Pinto made also a different one for the MiG-21, because the number of options would have grown too much.

**Comment by colingeniet** [comment]
Reading this table in more details, I feel like it might not be complete. A better source may be this table from the Speciell klargöringsinstruktion (SKI) AJS, AJSH, AJSF, Kap 2.10 --- pictures from Nikolai:

![IMG_0249](https://user-images.githubusercontent.com/23284595/79267653-d4974180-7e99-11ea-8621-9398e0838770.JPG)
![IMG_0250](https://user-images.githubusercontent.com/23284595/79267752-f4c70080-7e99-11ea-9723-51971c67c460.JPG)
![IMG_0251](https://user-images.githubusercontent.com/23284595/79267797-07413a00-7e9a-11ea-9188-89273f702721.JPG)
![IMG_0252](https://user-images.githubusercontent.com/23284595/79267812-0c9e8480-7e9a-11ea-8100-17c7f0193064.JPG)

It contains more loadouts, including some that I was surprised not to see in the first one, such as 2x RB15, or 4x RB75.

**Comment by NikolaiVChr** [comment]
At the very least it can carry this I think as in the AJ:

![37vapenalt](https://user-images.githubusercontent.com/3764194/79269959-8e43e180-7e9d-11ea-828c-4cd3709dba38.jpg)

**Comment by NikolaiVChr** [comment]
![AJ37_N2_3_stor](https://user-images.githubusercontent.com/3764194/79270015-a9165600-7e9d-11ea-9d35-7d05dd73711d.jpg)

**Comment by NikolaiVChr** [comment]
![AJSHF37_Bev-stor](https://user-images.githubusercontent.com/3764194/79270061-baf7f900-7e9d-11ea-88f8-5292dbb58f6d.jpg)

**Comment by NikolaiVChr** [comment]
![image](https://user-images.githubusercontent.com/3764194/79270749-f5ae6100-7e9e-11ea-8df9-adab0cd6e677.png)

**Comment by NikolaiVChr** [comment]
![image](https://user-images.githubusercontent.com/3764194/79270787-0b238b00-7e9f-11ea-8546-2c78526d06ce.png)

**Comment by colingeniet** [comment]
To avoid ambiguity, that last table is the one Vanosten mentioned.

**Comment by NikolaiVChr** [comment]
![image](https://user-images.githubusercontent.com/3764194/79270967-550c7100-7e9f-11ea-812c-158243caea10.png)

**Comment by vanosten** [comment]
I am not sure how many combinations you can show on the loadout panel (and I would restrict to only show war weapons, not training or simulation stuff and not ferry config of Rb04 on centre pylon). A compromise could be to Visually differentiate between the ones mentioned on page 188ff in Del 2 and the ones mentioned in SKI. There are only few differences.

**Comment by colingeniet** [comment]
I made the new loadout dialog: #117 
I did it the fun way, with some nasal code to generate the table in the dialog---result is that loadouts can be added by simply editing the lists at the beginning of `Nasal/payload/loadout.nas`.

For the JA, I kept the existing historical loadouts with rb71s, and added a couple of more lightweight made-up modern loadouts.

For the AJS, currently all the loadouts are from the SKI table. There are a few versatile ones, with combination of Rb05/75, AKAN and Rb24/74. The others respect the 'get in, fire all at once, get out' philosophy, with only one type of weapon, plus one or two Rb24/74.
I made up a M90 loadout based on the Rb15 one because I could not find any source for this.

Any suggestion to improve the current choice of loadouts? We can add 1, maybe 3 loadouts to the AJS (would bring the total to 18 or 20). I think more would start to be too much.
